cypress/e2e/22-stickies.cy.ts

```diff
@@ -0,0 +1,343 @@
+import { WorkflowPage as WorkflowPageClass } from '../pages/workflow';
+
+const workflowPage = new WorkflowPageClass();
+
+describe('Canvas Actions', () => {
+	beforeEach(() => {
+		cy.resetAll();
+		cy.skipSetup();
+		workflowPage.actions.visit();
+		cy.waitForLoad();
+
+		cy.window()
+		// @ts-ignore
+		.then(win => win.onBeforeUnload && win.removeEventListener('beforeunload', win.onBeforeUnload))
```

Comment:
We changed this recently to `onBeforeUnloadNodeView`. Maybe we could extract this to cy.command so there's a single source of truth

Reply:
updated now to the latest approach.